README.md

```diff
@@ -53,6 +53,7 @@ Here a configuration example:
 
 ```yaml
 m6_web_kafka:
+    services_name_prefix: 'custom_prefix'
```

Comment:
Maybe, we can add a comment to say that we have the possibility to customize the service name by using our own prefix with an optional parameter.